pyproject.toml

```diff
@@ -821,8 +821,7 @@ ignore = [
     "PLE0605",
 
     # temporarily disabled
-    "RET503",
-    "TRY301"
```

Comment:
I have moved these to individual `noqa`, to prevent new errors from creeping in.
These can be resolved by code-owners in subsequent PRs.